xyz.audiostellar.AudioStellar.yml

```diff
@@ -0,0 +1,344 @@
+app-id: xyz.audiostellar.AudioStellar
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: audiostellar
+finish-args:
+  # We need MIDI, Audio, USB, DRI, and realtime performances
+  - --device=all
+  
+  # We need X11, PulseAudio, JACK and Pipewire support
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0  
+  
+  # We are using the OSC protocol
+  - --share=network
+  
+  - --filesystem=home
+  # Possible removable devices
+  - --filesystem=/mnt
+  - --filesystem=/media
+  - --filesystem=/run/media
+  
+modules:
+# openFrameworks dependencies
+  - shared-modules/glu/glu-9.json
```

Comment:
the git submodule is missing

Reply:
it is in the flatpak shared-modules git: https://github.com/flathub/shared-modules/tree/master/glu

Reply:
done in the last commit